src/ch/imedias/rsccfx/model/xml/Supporter.java

```diff
@@ -23,7 +23,7 @@
   public Supporter() {
     this.description = "";
     this.address = "";
-    this.port = "";
+    this.port = 5500;
```

Comment:
I would not implement a default value in here (as this is an empty constructor, so the supporter should truly be empty), I think it would make more sense to include maybe a check before saving the data if the port was filled out and if not, to save "5500" as port